ssl/t1_lib.c

```diff
@@ -954,6 +954,9 @@ static const unsigned char tls12_sigalgs[] = {
         tlsext_sigalg(TLSEXT_hash_sha256)
         tlsext_sigalg(TLSEXT_hash_sha224)
         tlsext_sigalg(TLSEXT_hash_sha1)
+    TLSEXT_hash_gostr3411, TLSEXT_signature_gostr34102001,
+    TLSEXT_hash_gostr34112012_256, TLSEXT_signature_gostr34102012_256,
+    TLSEXT_hash_gostr34112012_512, TLSEXT_signature_gostr34102012_512
```

Comment:
Are they also NOT present if the GOST engine is not available? Also see `ssl_sig_tbl` and `ssl_md_tbl` in t1_trce.c. These need to be updated I think to get rid of the "UNKNOWN" in the trace output.